backend/typescript/rest/entityRoutes.ts

```diff
@@ -117,27 +127,35 @@ entityRouter.put(
 
 // ROLES: Admin
 /* Delete entity by id */
-entityRouter.delete("/:id", async (req, res) => {
-  const { id } = req.params;
+entityRouter.delete(
+  "/:id",
+  isAuthorizedByRole(new Set(["Admin"])),
+  async (req, res) => {
+    const { id } = req.params;
 
-  try {
-    const deletedId = await entityService.deleteEntity(id);
-    res.status(200).json({ id: deletedId });
-  } catch (e: unknown) {
-    res.status(500).send(getErrorMessage(e));
-  }
-});
+    try {
+      const deletedId = await entityService.deleteEntity(id);
+      res.status(200).json({ id: deletedId });
+    } catch (e: unknown) {
+      res.status(500).send(getErrorMessage(e));
+    }
+  },
+);
 
 // ROLES: Admin + CC
 /* Get file associated with entity by fileUUID */
-entityRouter.get("/files/:fileUUID", async (req, res) => {
-  const { fileUUID } = req.params;
-  try {
-    const fileURL = await fileStorageService.getFile(fileUUID);
-    res.status(200).json({ fileURL });
-  } catch (e: unknown) {
-    res.status(500).send(getErrorMessage(e));
-  }
-});
+entityRouter.get(
+  "/files/:fileUUID",
```

Comment:
@e-wai should we leave get get endpoints unprotected as parents will need to get the photo for the registrant experience?

Reply:
I'm not super familiar with our entity code, is there a reason we didn't just want to access a blob storage link directly (like they do with resumes)?

Reply:
it's all starter code so I am not sure how this was originally set up